requirements-wheels.txt

```diff
@@ -14,6 +14,7 @@ pydantic == 1.10.*
 git+https://github.com/fbcotter/py3nvml#egg=py3nvml
 PyYAML == 6.0
 pytz == 2023.3
+ruamel.yaml == 0.17.*
```

Comment:
Just curious, is there a reason pyyaml can't be used for this?

Reply:
> Just curious, is there a reason pyyaml can't be used for this?

pyyaml can't keep comments in yaml after modification